resources/lang/en/snippet_collections.json

```
@@ -169,24 +196,31 @@
         ["the feeling of flight"],
         ["a half-remembered promise"],
         ["a foreboding feeling"],
-        ["the embrace of a family"]
+        ["the embrace of a family"],
+        ["the ache of wounds"]
       ],
       "forest": [
+        ["the thrill of running across trees"]
 
       ],
       "plains": [
+        ["the freedom of the vast plains"]
 
       ],
       "beach": [
+        ["the vigour of the ocean"]
```

unfortunately we use american english

```suggestion
        ["the vigor of the ocean"]
```

i do wonder if vigor could be replaced, though. it sounds a little odd.. might just be me though

---

Changed to "the wrath of the ocean", could this work better instead?